virttest/utils_net.py

```diff
@@ -4441,9 +4442,10 @@ def create_linux_bridge_tmux(linux_bridge_name, iface_name=None, ignore_status=F
             raise exceptions.TestError("Create bridge fail as there is already interface named '%s' on the host "
                                        "and can not delete with error: %s" % (linux_bridge_name, o))
     if iface_name:
-        cmd = 'tmux -c "ip link add name {0} type bridge; ip link set {1} up;' \
-              ' ip link set {1} master {0}; ip link set {0} up;' \
-              ' pkill dhclient; sleep 6; dhclient {0}; ifconfig {1} 0"'.format(linux_bridge_name, iface_name)
+        shell_cmd = "ip link add name {0} type bridge; ip link set {1} up; ip link set {1} master {0}; ip link set {0} up; pkill dhclient; sleep 6; dhclient {0};".format(linux_bridge_name, iface_name)
```

Comment:
PEP 8 suggests lines should be limited to 79 characters. Could you please update this line?

Reply:
Updated, thanks.